app/javascript/angular/tests/_map.test.js

```diff
@@ -229,6 +229,19 @@ test('removeAllMarkers removes all markers', t => {
   t.end();
 });
 
+test('removeAllMarkers clears the clustering', t => {
+  const clearMarkers = sinon.spy()
+  const service = _map({})
+  service.clusterer = { clearMarkers }
```

Comment:
This is a bit hacky. Maybe extracting the "clusterer" to another file would make it more testable but not sure